lib/backup_restore/s3_backup_store.rb

```diff
@@ -0,0 +1,90 @@
+require_dependency "backup_restore/backup_store"
+require_dependency "s3_helper"
+
+module BackupRestore
+  class S3BackupStore < BackupStore
+    DOWNLOAD_URL_EXPIRES_AFTER_SECONDS = 15
```

Comment:
Might want to conditionally set (`||=`) these constants in order to prevent warnings when reloading.